posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,97 @@
+
+image::img/blog/starter_gradle_output.png[screen capture of the starter gradlew libertyStart output, align="center"]
+
+A starter project using Gradle is generated for you with the following files:
+
+image::img/blog/starter_gradle_directory_structure.png[screen capture of the starter gradle directory structure, align="center"]
+
+For more information on developing your application in dev mode using either Maven or Gradle, see the link:{url-prefix}/docs/latest/development-mode.html[dev mode documentation].
+
+For further help on getting started actually developing your application, see some of our link:{url-prefix}/guides/?search=microprofile&key=tag[Microprofile guides] and link:{url-prefix}/guides/?search=jakarta%20ee&key=tag[Jakarta EE guides].
+
+
+[#Alternatives]
+== Other ways to start using Open Liberty
+
+There are a few other ways to get started with Open Liberty if you prefer instead:
+
+[#Docker]
+=== Docker image
+
+You can pull in the official Open Liberty docker image from link:https://hub.docker.com/_/open-liberty[Docker Hub] and build your application image on top of the Open Liberty image. For more information, see our guide on using Docker with Open Liberty, link:{url-prefix}/guides/docker.html[Using Docker containers to develop microservices].
+
+=== Maven
+
+You can also use Maven to pull in Open Liberty as a dependency for your application. To get started using Open Liberty with Maven, add the following to your `pom.xml`, while specifying the version of Open Liberty that you want to use:
+
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>[21.0.0.8,)</version>
+    <type>zip</type>
+</dependency>
+----
+
+For more information on using Open Liberty and Maven, visit our link:{url-prefix}/guides/maven-intro.html[Maven Intro Guide].
+
+=== Gradle
+
+You can also use Gradle to pull in Open Liberty as a dependency for your application. To get started using Open Liberty with Gradle, add the following to `build.gradle`, while specifying the version of Open Liberty that you want to use:
+
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[21.0.0.8,)'
+}
+----
+
+For more information on using Open Liberty and Gradle, visit our link:{url-prefix}/guides/gradle-intro.html[Gradle Intro Guide].
+
+=== Open Liberty Binary Package
+
+Download an Open Liberty release from our link:{url-prefix}/downloads/#runtime_releases_table_container[Getting Started] page and extract the package. Then navigate to the `bin` directory and execute the `server create <server_name>` command to create an Open Liberty server, or run the `./server start` command to automatically create and start a `defaultServer`. Then, you can add in your application to the dropins directory, or add the path to your application in the newly created `<path_to_Open_Liberty_install>/servers/<server_name>/server.xml`.
```

Comment:
per Michals repsonse to my earlier comment- should we remove `./` from command example syntax, since it's OS dependent? Here and in the previous Gradle command example

Reply:
Right, sorry i thought i fixed it! I will update this in the next commit.